sdks/python/apache_beam/io/gcp/shared_test_variables.py

```diff
@@ -0,0 +1,63 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This file looks a bit awkward here.. also looking closer, the stub data in  `bigquery_file_loads_test`  is now also split across two files, which looks awkward too. A file like this would be better placed in /tests subdir in this folder, and could include all the necessary shared BQ stubs data & schema in one place.

Alternatively, we can reverse the course of my previous suggestion and redefine `_ELEMENTS` in `sdks/python/apache_beam/io/gcp/bigquery_test.py`. Since originally we have `_ELEMENTS = [elm[1] for elm in _DESTINATION_ELEMENT_PAIRS]` to avoid extra copypasting, it can be  redefined here as sth like: `[{'name': 'beam', 'language': 'py'}, {'name': 'beam', 'language': 'java'}, {'name': 'beam', 'language': 'go'}, {'name': 'flink', 'language': 'java'}, {'name': 'flink', 'language': 'scala'}, {'name': 'spark', 'language': 'scala'}, {'name': 'spark', 'language': 'py'}, {'name': 'spark', 'language': 'scala'}, {'name': 'beam', 'foundation': 'apache'}, {'name': 'flink', 'foundation': 'apache'}, {'name': 'spark', 'foundation': 'apache'}]`